quiz/models.py

```diff
@@ -51,15 +71,24 @@ def save(self, *args, **kwargs):
             except IntegrityError:
                 self.code = self.code_generator()
 
+
 class Contestant(models.Model):
-    # Contestant's room
+    """
+    Contestant model used to define a quiz contestant.
+    Each contestant has a handle (aka name), a score and a completed flag.
+    """
+    # Room name
     room = models.ForeignKey(Room, on_delete=models.CASCADE)
     # Handle i.e. username
     handle = models.CharField(max_length=200)
-    # user's score
+    # Score
     score = models.IntegerField(default=0)
     # has contestant completed
-    complete = models.IntegerField(default=1)
+    complete = models.BooleanField(default=False)
 
     def __str__(self):
-        return self.handle
\ No newline at end of file
+        return self.handle
+
+    def clean(self):
```

Comment:
Make sure Model clean does nothing

Reply:
The `clean()` method is provided to allow custom validation, see https://docs.djangoproject.com/en/1.11/ref/models/instances/#django.db.models.Model.clean